Xamarin.PropertyEditing.Mac/Controls/CollectionInlineEditorControl.cs

```diff
@@ -25,16 +28,30 @@ public CollectionInlineEditorControl (IHostResourceProvider hostResources)
 			};
 
 			this.openCollection.Activated += (o, e) => {
+
+				openCollection.Enabled = false;
```

Comment:
This causes odd behaviour in stand alone, so think it will affect native Cocoa applications that use proppy. Removing it does not appear to affect VS4M, when built from source. So I think we can remove it.

Reply:
fixed